src/shared/searchbar/searchbar.scss

```diff
@@ -8,5 +8,14 @@
     }
     .search-btn {
         width: 10%;
```

Comment:
Please change this to a fixed width, so that button doesn't scale down on low resolution screens

Reply:
We can keep scaling down as the button is big enough, but please add min-width and max-width